proposals/0024-opacity-micromaps.md

```diff
@@ -164,15 +181,15 @@ a reachable use of one of the new flags is encountered.
 A reachable use is one that is found by traversing AST from active entry and
 export functions, or from subobject declarations when compiling a library.
 Traversal will follow local function calls, as well as traversing referenced
-decls and initializers.
+decls (`DeclRef`s and `DeclRefExpr`s) and initializers.
 
 As an implementation detail, an attribute may be used on the new flag
 definitions, such as an existing Clang availability attribute or a new custom
-HLSL-specific attribute.
-
-AST traversal from entry points will traverse DeclRefs and initializers to
-detect the use of the new ray flag.  AST traversal will be added for subobject
-declarations on library targets to detect any use of the new pipeline flag.
+HLSL-specific attribute. Specifically, of the new flags introduced,
+`RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS`, 
+`RAY_FLAG_FORCE_OMM_2_STATE`, and `RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS`
+will have an availability attribute, which restricts their usage to
+shader model 6.9. `RAYQUERY_FLAG_NONE` will be left unrestricted.
 
 This will have implications for uses of these flags outside of the intended
 targets.  Since they are just uint values, it's technically legal to refer to
```

Comment:
I think we can use `DefaultError` here now, since this will be the only diagnostic used to catch use of these flags in an earlier shader model.